hack/start_flotta_k8s.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export OPERATOR_PATH="<path to flotta operator repo>"
+export DEVICE_WORKER_PATH="<path to flotta device worker repo>"
+export YGGDRASIL_PATH="<path to yggdrasil repo>"
+
+# Start flotta operator
+gnome-terminal --tab -- bash -c \
+        "cd $OPERATOR_PATH; \
+        IMG=<image registry and tag> make docker-build docker-push; \
+        IMG=<image registry and tag> TARGET=k8s make deploy; \
+        exec bash"
+
+# Start flotta device worker
+gnome-terminal --tab -- bash -c \
+        "cd $DEVICE_WORKER_PATH; \
+        make uninstall; \
+        make install; \
+        exec bash"
+
+# Start yggdrasil
+gnome-terminal --tab -- bash -c \
+        "cd $YGGDRASIL_PATH; \
+        git checkout cdde836c519ac72302714b5f7ae90e45de7c79a7; \
```

Comment:
Doubt here: can we checkout the HEAD? In this way the script will always be up-to-date.

In addition [here](https://github.com/project-flotta/flotta-operator/blob/f3906500370d89025ace44306439127cfa69bd15/docs/user-guide/running.md?plain=1#L70) there isn't a specific commit to checkout.

Reply:
If I remember correctly, after we started to use yggdrasil [here](https://github.com/RedHatInsights/yggdrasil), @eloycoto changed [here](https://github.com/project-flotta/flotta-device-worker/blob/main/README.md?plain=1#L54) to use this commit due to issues that came up. @eloycoto Can we checkout the HEAD or still need to use this commit?

In addition, @gciavarrini I think [this line](https://github.com/project-flotta/flotta-operator/blob/f3906500370d89025ace44306439127cfa69bd15/docs/user-guide/running.md?plain=1#L70) isn't updated this we use `RedHatInsights/yggdrasil` ([here](https://github.com/project-flotta/flotta-device-worker/blob/main/README.md?plain=1#L52))